### PR TITLE
Add cli commands to regen hooks & keys

### LINKED ISSUE #2491: Rewrite authorized_keys and hooks from command line (feature request)
- Gitea version (or commit ref): v1.2.0-rc2

## Description

Please add an option to rewrite authorized_keys and hooks from command line (same like already can be done from webui). With this, automated deployment of upgrades can achieved much more easy.

Thank you!

### PR DESCRIPTION
**Pull request by techknowlogick**
Rewrite authorized_keys and hooks from command line

Fix #2491

### CODE REVIEW

**Review by jonasfranz — CHANGES_REQUESTED** [review]

File: cmd/admin.go [hunk 1]
```diff
@@ -195,3 +216,19 @@ func getReleaseCount(id int64) (int64, error) {
 		},
 	)
 }
+
+func runRegenerateHooks(c *cli.Context) error {
+	err := models.SyncRepositoryHooks()
```
[the remark this answers is not in the document]
Comment (techknowlogick) [reply cmd/admin.go]: Thanks @JonasFranzDEV I've made this change, and the other change you requested.

**Comment by lunny** [comment]
LGTM

**Review by appleboy — APPROVED** [review]

**Comment by lafriks** [comment]
@techknowlogick please add this commands also to docs page (cli)

**Comment by lafriks** [comment]
@appleboy please do not merge while all reviewers has not approved, it this case probably it was not so important but it's better to follow this rule even if requested changes was fixed

**Comment by lunny** [comment]
yes @appleboy, @JonasFranzDEV didn't approve.

**Comment by techknowlogick** [comment]
@lafriks I've added these docs to https://docs.gitea.io/en-us/command-line/ already, is there another place that they need to be?

**Comment by lafriks** [comment]
@techknowlogick yes that what I mean, false alarm, all is good :)

**Pull request merged**